Dockerfile

```diff
@@ -1,18 +1,18 @@
-FROM alpine:3.4
+FROM mhart/alpine-node
 
-WORKDIR /usr/src/app
+RUN npm install -g reveal-md && \
+    npm cache clean --force && \
+    apk add --no-cache --update tini && \
+    rm -rf /var/cache/apk
+
+COPY resources/* /usr/src/app/
 
-RUN apk update && apk add nodejs
-RUN npm install -g reveal-md
-RUN mkdir -p /usr/lib/node_modules/reveal-md/node_modules/reveal.js/images
+ADD slides.md /usr/src/app/
```

Comment:
don't add the slides at build time, we want to be able to reuse this image for different presentations without rebuilding it